systers_portal/static/css/style.css

```diff
@@ -48,22 +48,35 @@ body > .container {
 /* Index page styles
 -------------------------------------------------- */
 .jumbotron-custom {
+  width: 100%;
   background-color: #fff;
   text-align: center;
-  border: 0;
+  border: 0;  
   box-shadow: none;
 }
 
-.featurette-divider {
-  margin: 4em 0;
+.featurette-divider {  
+  margin: 4em 0; 
 }
 
-.featurette-heading {
+.featurette-heading {  
   font-weight: 300;
   line-height: 1;
   letter-spacing: -1px;
   margin-top: 0;
-  font-size: 3.5em;
+  font-size: 3.5em;  
+}
+
+@media screen and (max-width: 991px){
+	img.img-responsive {
+		  display: block;
+		  margin-left: auto;
+		  margin-right: auto 
+	}
+
+	.index {
+		text-align: center;
```

Comment:
Thanks @Ana.  I'll apply the changes and do another PR.

On Wed, Dec 31, 2014 at 10:39 AM, Ana Balica <notifications@github.com>
wrote:

> In systers_portal/static/css/style.css
> <https://github.com/systers/portal/pull/58#discussion-diff-22377926>:
>
> >    font-weight: 300;
> >    line-height: 1;
> >    letter-spacing: -1px;
> >    margin-top: 0;
> > -  font-size: 3.5em;
> > +  font-size: 3.5em;
> > +}
> > +
> > +@media screen and (max-width: 991px){
> > +	img.img-responsive {
> > +		  display: block;
> > +		  margin-left: auto;
> > +		  margin-right: auto
> > +	}
> > +
> > +	.index {
> > +		text-align: center;
>
> So this style can be applied to a range of classes that include the
> featurette heading, body and the button. In fact, all those elements are in
> a div with featurette. So it can be done like that and remove the
> necessity to create another div:
>
> .featurette {
>   text-align: center;
> }
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/systers/portal/pull/58/files#r22377926>.
>